app/controllers/items/detail.js

```diff
@@ -644,6 +677,23 @@ export default GoodcityController.extend(
         this.send("saveItem", item);
       },
 
+      /**
+       *  Add the default suggested description for selected language
+       * @param {string} language - Language EN | Zh-TW
+       */
+      addDefaultDescriptionFor(language) {
+        const item = this.get("item");
+        if (language === "en") {
+          const description = this.get("item.code.descriptionEn");
+          item.set("notes", description);
+        } else {
+          const description = this.get("item.code.descriptionZhTw");
+          item.set("notesZhTw", description);
+        }
+        this.send("saveItem", item);
```

Comment:
```suggestion
        await this.send("saveItem", item);
```

I assume `saveItem` is asynchronous